src/SFA.DAS.EmployerRequestApprenticeTraining.Web/Models/EmployerRequest/TrainingOptions.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Diagnostics.CodeAnalysis;
+
+namespace SFA.DAS.EmployerRequestApprenticeTraining.Web.Models.EmployerRequest
+{
+    [ExcludeFromCodeCoverage]
+    public static class TrainingOptions
+    {
+        public const string AtApprenticesWorkplace = "At apprentice's workplace";
```

Comment:
Should we have these values in the config so that we can change them more easily if needed .  The same will apply to the provider side?